### PR TITLE
[DONOTMERGE] platform.mk: Kona-specific sensors

### PR DESCRIPTION
**Pull request by ix5**
**DONOTMERGE** only for bringup purposes
@kholk

### CODE REVIEW

File: platform.mk [hunk 1]
```diff
@@ -296,26 +296,49 @@ PRODUCT_PACKAGES += \
...
-    msmnile_qrd_ak991x_0.json \
-    msmnile_qrd_lsm6dso_0.json \
-    msmnile_qrd_tmd2725.json \
-    msmnile_shtw2_0.json \
-    msmnile_tmd2725.json \
-    msmnile_tmd3725.json
+    kona_ak991x_0.json \
+    kona_ak991x_0_somc_platform.json \
+    kona_ak991x_0_somc_product.json \
+    kona_amd_sw_disabled.json \
```
Comment (MarijnS95) [inline platform.mk:302]: This configuration already exists in `sns_amd_sw_disabled.json` (included above), albeit with more `soc_id`s (including `"356"` from `kona_amd_sw_disabled.json`).

File: platform.mk [hunk 2]
```diff
@@ -296,26 +296,49 @@ PRODUCT_PACKAGES += \
...
+PRODUCT_PACKAGES += \
+    sns_ccd_v2_walk.json \
+    sns_ccd_v3_1_walk.json \
+    sns_ccd_v3_walk.json \
+    sns_device_orient_somc_platform.json \
+    sns_fmv_legacy.json \
+    sns_heart_rate.json \
+    sns_mag_cal_legacy.json \
+    sns_wrist_pedo.json \
+    wigig_sensing_0.json
```
Comment (MarijnS95) [inline platform.mk:341]: These all sound pretty generic though, albeit they were only introduced with Kona...

**Comment by MarijnS95** [comment]
All sensors appear to be fine on-device except proximity, that only changes between 0 and 5cm. Perhaps this is a binary sensor?

File: platform.mk (hunk 2 above)
Comment (jerpelea) [reply platform.mk:341]: they should go to common !

Comment (ix5) [reply platform.mk:341]: No other platforms have pulled these in yet. The sensor files themselves are in common.

**Comment by MarijnS95** [comment]
@ix5 Forgot to write yesterday that stock seems to include `stk3x3x_0.json` as well. Other than that all files appear to be there.

**Comment by jerpelea** [comment]
@ix5 it looks good 
please add stk3x3x_0.json so that I can merge it

**Comment by ix5** [comment]
> @ix5 Forgot to write yesterday that stock seems to include `stk3x3x_0.json` as well. Other than that all files appear to be there.

That's another one that is duplicated with a `kona_` prefix :-/ How do we want to proceed? Pull in the generic one or kona?

**Comment by MarijnS95** [comment]
@ix5 It appears all Kona files have SoC ID `356`, the non-prefixed one doesn't have it. Seems like stock "include it all" policy despite being unused, perhaps?

**Comment by ix5** [comment]
@jerpelea please reopen so I can change target branch

**Comment by jerpelea** [comment]
I will cherry-pick

**Pull request closed without merging**